src/en/charms-storage.md

```diff
@@ -304,6 +305,15 @@ config:
   security.privileged: "true"
 ```
 
+### Remove storage
+
+Specify one or more unit storage, volume, or filesystem IDs, which can be found
+in the output of `juju storage`, like this:
+
+```bash
+juju remove-storage pgdata/0
+```
+
```

Comment:
Are there any caveats for this? What happens if I remove storage from a running charm? Does the storage automatically get removed if I remove the charm that's using it?